### PR TITLE
Bump grafana

### PR DESCRIPTION
**Pull request by brian-avery**
This is a 1.5/1.6 version of https://github.com/istio/istio/pull/24397. 

That version updates to Grafana 7.0.2. Instead, we're updating to 6.7.4 from 6.5.2. I have run through https://istio.io/latest/docs/tasks/observability/metrics/using-istio-dashboard/ and it worked. Note that the logo is broken, but that appears to be because https://istio.io/img/istio-logo.svg no longer exists. 

image

image

This fixes CVE-2020-13379: https://grafana.com/blog/2020/06/03/grafana-6.7.4-and-7.0.2-released-with-important-security-fix/

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[x] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: manifests/charts/istiod-remote/files/gen-istiod-remote.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: istio-sidecar-injector
-  namespace: default
+  namespace: opentlc-mgr-smcp
```
Comment (douglas-reid) [inline manifests/charts/istiod-remote/files/gen-istiod-remote.yaml:7]: why is this change a part of this PR ?

**Comment by douglas-reid** [comment]
Can we somehow restore the logo?  @frankbu -- any idea why it was removed?

File: manifests/charts/istiod-remote/files/gen-istiod-remote.yaml (hunk 1 above)
Comment (brian-avery) [reply manifests/charts/istiod-remote/files/gen-istiod-remote.yaml]: This change came from my environment. I backed it out. That said, the change wasn't in my environment variables or anywhere else in the codebase other than the output of `make gen`. I need to dig in to see why this is occurring.

**Comment by brian-avery** [comment]
I think the removal of this image was a result of the moves to /latest urls on istio.io.

**Comment by ericvn** [comment]
I did create a PR: https://github.com/istio/istio.io/pull/7541 which should fix the img redirects. It works using the netlify generated in the PR, but I need someone to review and verify my logic is correct.

**Pull request closed without merging**